scripts/protocgen.sh

```diff
@@ -30,5 +30,6 @@ Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types:. \
 done
 
 ## move proto files to the right places
-cp -r github.com/medibloc/panacea-core/* ./
-rm -rf github.com
+find ./x -type f -name "*.pb*.go" -exec rm {} \;
+cp -rv github.com/medibloc/panacea-core/v*/x/* ./x/
```

Comment:
The go_package of the proto file was modified to include v2, but v2 is not included in the actual go folder structure.